ansible/roles/ci/code_style_check/license/files/gather_missing_licences.py

```diff
@@ -47,7 +47,6 @@ def gather_arguments():
 
     with open('/tmp/git_source', 'r') as tmp_file:
         source = tmp_file.read().strip()
-    print("\n", source)
```

Comment:
How come and you remove this now?

Reply:
I didn't see it in the last PR, I thought you where talking about the other print I had altered